jsapp/js/components/submissions/tableConstants.ts

```diff
@@ -59,21 +61,41 @@ CELLS_WIDTH_OVERRIDES[VALIDATION_STATUS_ID_PROP] = 125;
 CELLS_WIDTH_OVERRIDES[META_QUESTION_TYPES.start] = 110;
 CELLS_WIDTH_OVERRIDES[META_QUESTION_TYPES.end] = 110;
 CELLS_WIDTH_OVERRIDES[ADDITIONAL_SUBMISSION_PROPS._id] = 100;
-CELLS_WIDTH_OVERRIDES[QUESTION_TYPES.image.id] = 110;
-CELLS_WIDTH_OVERRIDES[QUESTION_TYPES.audio.id] = 170;
-CELLS_WIDTH_OVERRIDES[QUESTION_TYPES.video.id] = 110;
+CELLS_WIDTH_OVERRIDES[QuestionTypeName.image] = 110;
+CELLS_WIDTH_OVERRIDES[QuestionTypeName.audio] = 170;
+CELLS_WIDTH_OVERRIDES[QuestionTypeName.video] = 110;
 CELLS_WIDTH_OVERRIDES[META_QUESTION_TYPES['background-audio']] = 170;
 Object.freeze(CELLS_WIDTH_OVERRIDES);
 
+/**
+ * For these question types the UI will display a dropdown filter in Data Table
+ * for the matching column.
+ */
+export const DROPDOWN_FILTER_QUESTION_TYPES = [
+  QuestionTypeName.select_multiple,
+  QuestionTypeName.select_one,
+];
+
+/**
+ * For these question types the UI will display a text filter in Data Table for
+ * the matching column.
+ */
 export const TEXT_FILTER_QUESTION_TYPES = [
-  QUESTION_TYPES.text.id,
-  QUESTION_TYPES.integer.id,
-  QUESTION_TYPES.decimal.id,
-  QUESTION_TYPES.date.id,
-  QUESTION_TYPES.time.id,
-  QUESTION_TYPES.datetime.id,
-  QUESTION_TYPES.barcode.id,
-  QUESTION_TYPES.calculate.id,
+  QuestionTypeName.barcode,
+  QuestionTypeName.calculate,
+  QuestionTypeName.date,
+  QuestionTypeName.datetime,
+  QuestionTypeName.decimal,
+  QuestionTypeName.integer,
+  QuestionTypeName.range,
```

Comment:
It looks like `range`, `rank`, and `score` got reversed somehow. Here they are on beta:
![image](https://github.com/kobotoolbox/kpi/assets/16762675/edd59bd3-42b4-4c88-8ed9-142e6b344a45)
`range` cannot be filtered but `rank` and `score` can

And on the branch:
![image](https://github.com/kobotoolbox/kpi/assets/16762675/00301b99-c04f-4fbb-a8f1-775ec81f5f39)
`range` can be filtered but `rank` and `score` cannot